datasets/my/investor_alert_list/crawler.py

```diff
@@ -0,0 +1,56 @@
+import json
+from zavod import Context, helpers as h
+
+
+def crawl_item(input_dict: dict, context: Context):
+
+    name = input_dict.pop("name")
+
+    # If it's a potential clone, we remove the "potential clone" from the name
+    name = name.replace("Potential clone entity – ", "")
+
+    entity = context.make("LegalEntity")
+    entity.id = context.make_id(name)
+    entity.add("topics", "sanction")
+    entity.add("name", name)
+    entity.add("address", input_dict.pop("address"))
+    entity.add("notes", input_dict.pop("remark"))
+
+    for website in input_dict.pop("website").split(" | "):
+        entity.add("sourceUrl", website)
```

Comment:
I would put these on the `website` property rather than `sourceUrl` - I've probably guided incorrectly on this in the past.